src/Sulu/Bundle/MediaBundle/Resources/public/js/components/collections/edit/files/main.js

```diff
@@ -264,7 +249,7 @@ define(function() {
          */
         renderSelectCollection: function() {
             this.sandbox.start([{
-                name: 'collections/components/collection-select@sulumedia',
+                name: 'collections/overlays/collection-select@sulumedia',
```

Comment:
I don't know if I would put the overlays in a separate folder... Could be a bit confusing, since next to it are only components. /cc @chirimoya  @wachterjohannes